website/docs/reference/appsmith-framework/console-object.md

```diff
@@ -1,223 +1,73 @@
 ---
 sidebar_position: 2
+description: The console object provides a way to log messages to the logs tab.
 ---
 
-# Console Object
+# console
 
-The [console object](https://developer.mozilla.org/en-US/docs/Web/API/Console\_API)provides an easy way to send logging messages from the browser to the development console or to display messages in the browser when an error occurs. By default, console output would appear in the browser's console tab, which you can view by invoking your browser's developer tools.
+The `console` object provides a way to log messages to the Logs tab, aiding in code debugging and monitoring. This page gives comprehensive information about the console object and its methods.
 
-The console is integral to any developer's toolkit - it allows you to monitor what your program is doing by logging messages, errors, and warnings as they occur. These informative logs make it much easier to debug your code and locate the source of errors and unexpected behavior.
-
-Appsmith provides the global console object for logging information about your API, Queries, and Widgets properties in your JavaScript code. Invoke a console object using the mustache sign `{{}}` in widget properties or directly in your code.
-
-:::note
-The console logs **is not** saved and are **only** available for the **current session.**
-:::
 
 ## Methods
+The methods log the messages to the logs tab. The logged messages are only accessible during the ongoing session. 
```

Comment:
```suggestion
The methods log the messages in the **Logs** tab of the Javascript editor. The messages are not saved and are only available for the current session. 
```

Reply:
The Logs tab is available for database queries, JavaScript editor, and also for API queries.

Reply:
updated to add this, and modified the image.